scripts/sanity_chk/harness.py

```diff
@@ -41,6 +48,10 @@ def configure(self, instance):
             self.repeat = config.get('repeat', 1)
             self.ordered = config.get('ordered', True)
             self.record = config.get('record', {})
+            self.running_dir = config.get('running_dir', None)
+            self.source_dir = config.get('source_dir', None)
+            self.pytest_root = config.get('pytest_root', 'pytest')
+            self.pytest_timeout = config.get('pytest_timeout', 'pytest')
```

Comment:
copy-paste error? -> config.get('pytest_timeout', 15) ?

Reply:
Fixed.
And the example commit is included in this PR